server/kit/kitserver.go

```diff
@@ -87,29 +88,39 @@ func NewServer(svc Service) *Server {
 		errs  *errorreporting.Client
 		propr propagation.HTTPFormat
 	)
-	// if in App Engine, initiate an error reporter and the stackdriver exporters
-	if isGAE() {
-		projectID, serviceID, svcVersion := getGAEInfo()
-		errs, err = errorreporting.NewClient(ctx, projectID, errorreporting.Config{
-			ServiceName:    serviceID,
-			ServiceVersion: svcVersion,
-			OnError: func(err error) {
-				lg.Log("error", err,
-					"message", "error reporting client encountered an error")
-			},
-		})
-		if err != nil {
-			lg.Log("error", err,
-				"message", "unable to initiate error reporting client")
-		}
 
-		err = initSDExporter(projectID, serviceID, svcVersion, lg)
-		if err != nil {
-			lg.Log("error", err,
-				"message", "unable to initiate error tracing exporter")
+	projectID := googleProjectID()
+	if projectID != "" {
```

Comment:
We can drop this `if` check, right? The nested code makes all the checks it they need.

Reply:
hm...